convox/rack
# Feat 651 wildcard domain

### What is the feature/fix?

https://github.com/convox/issues-private/issues/651

Enable wildcard domain for review app.

Alb has limit to add 25 cert at most.

This feature creates a wildcard cert for the review app and add the certificate to the loadbalancer


### Add screenshot or video (optional)
[
** Any screenshot or video capture using the feature **](https://user-images.githubusercontent.com/12232198/196509713-70acd640-9959-49be-89ba-4ee56a61a3f8.mp4)

### Does it has a breaking change?
no

### How to use/test it?

It has serveral dependent PR:

Deploy console app containing these pr changes.
You should see in the review work flow there is a check for that.
Then create a review workflow and deploy a review app.

### Checklist
